code/modules/projectiles/projectile/magic.dm

```diff
@@ -169,6 +169,9 @@
 	M.invisibility = INVISIBILITY_ABSTRACT
 
 	var/list/contents = M.contents.Copy()
```

Comment:
can probably just be get_equipped_items too since everything gets equip_to_appropriate_slot called as well.